.gdbinit

```diff
@@ -0,0 +1,7 @@
+# Disable signal interrupts from x86 interrupts
+# SIGRTMIN + INTERRUPT_PRIORITY_LOW
+handle SIG35 nostop noprint
+# SIGRTMIN + INTERRUPT_PRIORITY_NORMAL
+handle SIG36 nostop noprint
+# SIGRTMIN + INTERRUPT_PRIORITY_HIGN
+handle SIG37 nostop noprint
```

Comment:
IIRC I also added SIG38 for disable/enable interrupts.

Reply:
Ah right, forgot about this one

```c
// Add a rule for the masking (critical sectioning) of the signals.
act.sa_mask = block_mask;
act.sa_sigaction = prv_sig_state_handler;
sigaction(SIGRTMIN + NUM_INTERRUPT_PRIORITIES, &act, NULL);
```